clippy_lints/src/string_to_string.rs

```diff
@@ -0,0 +1,57 @@
+use crate::utils::is_type_diagnostic_item;
+use crate::utils::span_lint_and_help;
+
+use if_chain::if_chain;
+use rustc_hir as hir;
+use rustc_hir::*;
```

Comment:
same NIT here for imports.
Don't forget to test again to add back some `hir::` in your code if necessary.

```suggestion
use crate::utils::{is_type_diagnostic_item, span_lint_and_help};
use if_chain::if_chain;
use rustc_hir as hir;
```